src/store.ts

```diff
@@ -663,33 +658,6 @@ const createMlStore = (logging: Logging) => {
             });
           },
 
-          /**
-           * Generally project loads go via MakeCode as it reads the hex but when we open projects
-           * from microbit.org we have the JSON already and use this route.
-           */
-          loadProject(project: MakeCodeProject, name: string) {
-            const newActions = getActionsFromProject(project);
-            set(({ settings }) => {
-              const timestamp = Date.now();
-              return {
-                settings: {
-                  ...settings,
-                  toursCompleted: Array.from(
-                    new Set([...settings.toursCompleted, "DataSamplesRecorded"])
-                  ),
-                },
-                actions: newActions,
-                dataWindow: getDataWindowFromActions(newActions),
-                model: undefined,
-                project: renameProject(project, name),
-                projectEdited: true,
-                appEditNeedsFlushToEditor: true,
-                timestamp,
-                // We don't update projectLoadTimestamp here as we don't want a toast notification for .org import
```

Comment:
This has changed... I don't think this difference is really significant but welcome views as to whether I'm missing something.

Reply:
Nope, don't see why we don't want it really.